packages/mediapipe-core/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: mediapipe_core
+description: A starting point for Dart libraries or applications.
+version: 1.0.0
+
+repository: https://github.com/google/flutter-mediapipe
```

Comment:
Is there an issue tracker? In flutter/packages we usually include an `issue_tracker`